README.md

```diff
@@ -37,39 +37,21 @@ git clone git@github.com:fedimint/minimint.git
 cd minimint
 ```
 
-### Building and Configuring
-First we need to build the executables, set env vars, and generate mint configs within the Nix environment (takes a few minutes the first time):
+### Setting up the federation
+Just run the following script
 ```shell
-nix-shell
-source ./scripts/build.sh [fed_size] [dir]
+./scripts/tmuxinator.sh
```

Comment:
When running I get the error
```
WARNING: You are running tmuxinator with an unsupported version of tmux.
```

Perhaps there is a missing prerequisite or should we run within `nix-shell`?

Reply:
Fixed that and ran it in an empty container for testing (my previous test env *only* had tmux and I forgot about that fact). I also added `bc`, apparently `pegin.sh` depends on that now.